dialogue-core/src/main/java/com/palantir/dialogue/core/ContentDecodingChannel.java

```diff
@@ -98,9 +102,6 @@ private static Response decompress(Response input) {
 
         @Override
         public InputStream body() {
-            if (body == null) {
-                delegate.body();
-            }
```

Comment:
:man_facepalming: I must have had a bad re-stack on the content-decoding change, sorry!